docs/source/gbd2017_models/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -307,25 +318,84 @@ Risk Factor Hierarchy
 
 .. image:: iron_risk_hierarchy.svg
 
-Cause Model Diagram
--------------------
 
-Data Description
-----------------
+Modeling Strategy
```

Comment:
Is this 'Vivarium Modeling Strategy'?

Reply:
Yes! I updated the headings/structure to make this more clear. Thanks :)